packages/gatsby-plugin-page-creator/src/__tests__/derive-path.ts

```diff
@@ -1,105 +1,209 @@
 import { derivePath } from "../derive-path"
 import reporter from "gatsby-cli/lib/reporter"
-import { createPath } from "gatsby-page-utils"
 
 describe(`derive-path`, () => {
   it(`has basic support`, () => {
     expect(
-      derivePath(`product/{Product.id}.js`, { id: `1` }, reporter)
+      derivePath(`product/{Product.id}`, { id: `1` }, reporter).derivedPath
```

Comment:
The different shape comes from the changes in `derivePath` where `errors` is also exposed. Should I also test that?

Another note: I removed all the file extensions to keep the tests more focused on the URL. Some examples with file extension are at the bottom